packages/dina-ui/components/object-store/metadata/MetadataUpload.tsx

```diff
@@ -42,25 +42,8 @@ export function MetadataUpload({ buttonBar }: MetadataUploadProps) {
         }
       );
     }
-    // Set default values for the new Metadatas:
-    const {
-      data: { values: defaultValues }
-    } = await apiClient.get<{ values: DefaultValue[] }>(
-      "objectstore-api/config/default-values",
-      {}
-    );
-    const metadataDefaults: Partial<Metadata> = {
-      publiclyReleasable: true
-    };
-    for (const defaultValue of defaultValues.filter(
-      ({ type }) => type === "metadata"
-    )) {
-      metadataDefaults[defaultValue.attribute as keyof Metadata] =
-        defaultValue.value as any;
-    }
```

Comment:
This part will need to be added back (except the publiclyReleasable part) so the License and Copyright information is pre-populated when the form is loaded (for new records)